### PR TITLE
Add info to document page

### PR DESCRIPTION
**Pull request by jchristensonCHI**
This branch has been fixed and can be pushed to main once approved!

### CODE REVIEW

File: civiclens/templates/document.html [hunk 1]
```diff
@@ -114,67 +118,124 @@ <h6 class="font-italic font-weight-normal">Original title: {{doc.title}}</h6>
...
-        <div class="card p-0">
-          <div class="card-body">
-            <p class="card-text p-2">comment.text...</p>
+        {% if comments_nlp.rep_comments %}
+          <span class="h6">{{ comments_nlp.num_topics }} topics emerged from {{ doc.comment_count }} total comments.</span>
+          <p></p>
+          <span class="p">[methods here...]</span>
+          <span class="p">[methods here...]</span>
+          <span class="p">[methods here...]</span>
+          <div class="container-fluid p-1 h-25" style="background-color: #FDFFF7;" id="chart">
```
Comment (jchristensonCHI) [inline civiclens/templates/document.html:138]: Lines 138-150 are where the graphs are included

**Review by jchristensonCHI — COMMENTED** [review]
Where the graph is added

File: civiclens/regulations/static/js/graph.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+        // title
+    svg.append("text")
+        .attr("x", 0)
+        .attr("y", 0)
+        .attr("text-anchor", "start")
+        .text("Total comments")
+        .attr("font-size", 14)
+        .attr("font-weight", "bold");
+}
```
Comment (michplunkett) [inline civiclens/regulations/static/js/graph.js:119]: It looks like the end of line check isn't working, @abejburton.

File: civiclens/regulations/static/js/graph.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
+// graph.js
+
+export function drawGraph() {
```
Comment (michplunkett) [inline civiclens/regulations/static/js/graph.js:3]: This function name should be more descriptive and shouldn't be the same as the one in the other file.

File: civiclens/regulations/static/js/two_graph.js [hunk 4]
```diff
@@ -0,0 +1,192 @@
+// graph.js
```
Comment (michplunkett) [inline civiclens/regulations/static/js/two_graph.js:1]: This isn't correct, though I don't think it's really needed either.

File: civiclens/templates/search_results.html [hunk 5]
```diff
@@ -67,57 +63,55 @@
...
               <div class="form-check">
-                <input class="form-check-input" type="checkbox" value="true" id="rule" name="rule" checked>
+                <input class="form-check-input" type="checkbox" name="rule" id="rule" value="Rule" checked>
                 <label class="form-check-label" for="rule">
                   Rule
                 </label>
               </div>
               <div class="form-check">
-                <input class="form-check-input" type="checkbox" value="true" id="proposed_rule" name="proposed_rule" checked>
+                <input class="form-check-input" type="checkbox" name="proposed_rule" id="Proposed Rule" value="Proposed Rule" checked>
```
Comment (michplunkett) [inline civiclens/templates/search_results.html:96]: All classes and ids should be in kebab-case (i.e., proposed-rule).

File: civiclens/templates/layouts/base.html [hunk 6]
```diff
@@ -16,14 +16,11 @@
     <link rel="icon" type="image/icon" href="/static/assets/img/favicon.ico" />
   </head>
   <body style="background-color:#FDFFF7">
-  </body>
-</html>
-
-<html>
```
Comment (michplunkett) [inline civiclens/templates/layouts/base.html:22]: Unless I'm missing something, all content should be in the body.

**Comment by michplunkett** [comment]
To get a better view of the problem you're running into, could you send me the console logs from your browser? The ones in your browser console.

**Review by claireboyd — APPROVED** [review]
LGTM!

**Pull request merged**